docker-compose.yaml

```diff
@@ -4,23 +4,17 @@ services:
 
   caddy:
     container_name: caddy
-    image: abiosoft/caddy:1.0.3-no-stats
-    ports:
-      - 80:80
-      - 443:443
+    image: caddy/caddy:2-alpine
     network_mode: host
-    command: -email ${LETSENCRYPT_EMAIL} -agree=${LETSENCRYPT_AGREE} -log stdout -host ${SEARX_HOSTNAME} -conf /etc/Caddyfile
+    command: caddy run -config /etc/caddy/Caddyfile
```

Comment:
why overriding the default command? the default Dockerfile already include the correct command: https://github.com/caddyserver/caddy-docker/blob/master/2.1/alpine/Dockerfile#L65

Reply:
removed.
(the other docker image seems to require this).